bumble/rfcomm.py

```diff
@@ -466,17 +468,30 @@ def __init__(
         self.state = DLC.State.INIT
         self.role = multiplexer.role
         self.c_r = 1 if self.role == Multiplexer.Role.INITIATOR else 0
-        self.sink = None
         self.connection_result = None
         self.drained = asyncio.Event()
         self.drained.set()
+        # Queued packets when sink is not set.
+        self._packet_queue = asyncio.Queue(maxsize=32)
```

Comment:
It doesn't seem that you need an async queue here, since no one if awaiting its elements. A simple `dequeue` would work just fine.

Reply:
Done